en/app/bluetooth-printers.md

```diff
@@ -2,7 +2,7 @@
 lang: en
 title: Connecting a Bluetooth printer
 ref: app-bluetooth-printers
-parent: Application
```

Comment:
For it to work, you also need to change the `title` attribute in `en/app/index.md`